efficient-eagles/early_internet/main/views.py

```diff
@@ -164,3 +170,14 @@ def get(self, request, q):
     def post(self, request, q):
         query = request.POST["query"]
         return redirect(reverse("search", args=[query]))
+
+
+class UserView(TemplateView):
+    template_name = 'profile.html'
+
+    def get(self, request, *args, **kwargs):
+        user = get_object_or_404(CustomUser, username=kwargs['user'])
```

Comment:
user and user_id are the same

Reply:
It would be better to go to /user/<the username> instead of /user/<the id of the user> for someone using the app

Reply:
> 
> 
> It would be better to go to /user/the_username instead of /user/id for someone using the app

Yes, but I'm talking about a variable - user and user_id being an exactly the same CustomUser instance